src/enzo/bondi_alpha.C

```diff
@@ -0,0 +1,56 @@
+#include "preincludes.h"
+#include "macros_and_parameters.h"
+
+
+/* Routine to return alpha, defined as rho/rho_inf, for a critical
+   Bondi accretion solution.  The argument is x = r / rBondiHoyle. 
+   Adapted from Orion, courtesy of Mark Krumholz */
```

Comment:
Does this follow the approach described in Krumholz et al. 2004(https://ui.adsabs.harvard.edu/abs/2004ApJ...611..399K)? It would be helpful to include a link to a description of the equations if it's easily available.

Reply:
Yes exactly. I've included a link in the file now.